api-gateway-config/scripts/lua/routing.lua

```diff
@@ -59,10 +59,15 @@ function _M.processCall()
       end
       -- Parse backend url
       local u = url.parse(opFields.backendUrl)
+      -- add http:// if no protocol is specified
+      if u.scheme == nil then
+        u = url.parse(utils.concatStrings({'http://', opFields.backendUrl}))
+      end
       ngx.req.set_uri(getUriPath(u.path))
       ngx.var.backendUrl = opFields.backendUrl
```

Comment:
@codymwalker yup I saw that as well. You're right the only other place `ngx.var.backendUrl` is being used is in mapping, and it doesn't need the scheme. It's being used there to parse out the query (not sure if it's being done the best way, but that's probably for another PR..). I tested out the query mappings and it seems to work fine.